src/TitleDetails/AddPieceModal/AddPieceModalContainer.js

```diff
@@ -11,12 +14,28 @@ import AddPieceModal from './AddPieceModal';
 
 const AddPieceModalContainer = ({
   close,
-  onSubmit,
   initialValues,
   instanceId,
+  mutator,
   onCheckIn,
+  onSubmit,
   poLine,
 }) => {
+  const [locations, setLocations] = useState();
+  const pieceLocation = initialValues.locationId;
+  const poLineLocations = poLine?.locations?.map(({ locationId }) => locationId) || [];
```

Comment:
the same, it's ids, not objects. Name of var should reflect what it contains, I hope you agree with that.

Reply:
Yes, renamed them